test/tests.js

```diff
@@ -1 +1 @@
-import './example.test.js';
+import './utils.test.js';
```

Comment:
you will need to make a file called utils.test.js. (or just replace with example.test.js and use that file since it already exists). This is not the same file as your utils.js.